esxi/tests/test_unit.py

```diff
@@ -286,3 +286,46 @@ def test_use_guest_hostname(vcsim_instance, dd_run_check, aggregator):
     aggregator.assert_metric("esxi.cpu.usage.avg", value=18, hostname="testing-vm")
     aggregator.assert_metric("esxi.cpu.usage.avg", value=19, hostname="test-vm-2")
     aggregator.assert_metric("esxi.cpu.usage.avg", value=26, hostname="localhost.localdomain")
+
+
+@pytest.mark.parametrize(
+    'excluded_tags',
+    [
+        pytest.param([], id="No excluded tags"),
+        pytest.param(['esxi_type'], id="type"),
+        pytest.param(['test'], id="unknown tag"),
```

Comment:
Maybe we could log a warning if the given `excluded_host_tags` contains tags other than the expected tags:`esxi_url`, `esxi_type`, `esxi_host`, `esxi_folder`, `esxi_cluster`, `esxi_compute`, `esxi_datacenter`, `esxi_datastore` ?

Reply:
Added some validation!